.github/workflows/release.yml

```diff
@@ -34,6 +34,39 @@ jobs:
         run: |
           VERSION=$(cargo tree -i wasmvm | grep -oE "[0-9]+(\.[0-9]+){2}-[0-9]+(\.[0-9]+){2}")
             echo ::set-output name=version::v$VERSION
+
+  build_static_lib:
+    name: Build static libraries
+    needs:
+      - get-version
+    if: ${{ needs.get-version.outputs.package-version != needs.get-version.outputs.latest-tag }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: set up
+        uses: actions/setup-go@v2
```

Comment:
Also use v2. The latest version is 3.2.1

Reply:
fixed